ADDING_PROJECTS.md

```diff
@@ -0,0 +1,222 @@
+- Specific build steps
+
+### Dockerfile Structure
+
+If your project needs a custom Dockerfile, follow this exact structure:
+
+```dockerfile
+ARG ZENML_VERSION=latest
+ARG PROJECT_DIR_NAME
+ARG EXTENSION_VERSION=0.1.6
+
+FROM zenmldocker/zenml-codespace:${ZENML_VERSION}
+
+# Set build arguments again for use in subsequent commands
+ARG PROJECT_DIR_NAME
+ARG EXTENSION_VERSION
+
+# Set the working directory for the project
+WORKDIR /home/coder/extensions/zenml-io.zenml-tutorial-${EXTENSION_VERSION}-universal/pipelines
+
+# Copy the specific project's requirements file
+COPY ./${PROJECT_DIR_NAME}/requirements.txt /tmp/requirements.txt
+
+# Install project-specific dependencies using uv for faster installation
+RUN uv pip install --system --no-cache -r /tmp/requirements.txt && \
+    rm /tmp/requirements.txt
+
+# Enable tutorial content for this specific project
+ENV ZENML_ENABLE_TUTORIAL=true
+
+# Default command can be overridden in docker run or docker-compose
+# CMD ["python", "run.py"] # Or your project's typical entrypoint
+```
+
+### When No Dockerfile is Needed
+
+If your project only requires Python dependencies listed in `requirements.txt`, **do not include a Dockerfile**. The projects backend will automatically build your project using the generic Dockerfile available at:
+[https://github.com/zenml-io/zenml-projects-backend/blob/main/.docker/project.Dockerfile](https://github.com/zenml-io/zenml-projects-backend/blob/main/.docker/project.Dockerfile)
+
+## 🔧 Backend Integration
+
+### YAML Configuration File
+
+Once your project is added to this repository, you **must** create a corresponding YAML configuration file in the `zenml-projects-backend` repository. This file should include all the following attributes:
+
+```yaml
+project_id: your-project-id
+name: Your Project Name
+description: A comprehensive description of what your project does and its key features
+github: https://github.com/zenml-io/zenml-projects/tree/main/your-project-directory
+preview_image: https://github.com/your-org/your-project/raw/main/assets/preview.png
+main_image_link: https://public-flavor-logos.s3.eu-central-1.amazonaws.com/projects/[PROJECT_NUMBER].jpg
+details: |
+  Detailed description of your project including:
+  
+  ### What It Does
+  
+  Explain the main functionality and purpose of your project
+  
+  ### Key Features
+  
+  - Feature 1: Description
+  - Feature 2: Description
+  - Feature 3: Description
+  
+  ### How It Works
+  
+  Step-by-step explanation of how users can use your project
+  
+  ### Architecture
+  
+  Brief overview of the technical architecture
+
+environment_variables:
+  CUSTOM_VAR: 'value'
+  ANOTHER_VAR: 'another_value'
+
+stack:
+  orchestrator: default
+  artifact_store: default
+  # Add other stack components if needed
+
+tags:
+- Domain (e.g., LLMOps, Computer Vision, MLOps)
+- Technology (e.g., PyTorch, TensorFlow, Hugging Face)
+- Use Case (e.g., Classification, Forecasting, NLP)
```

Comment:
shouldn't these be lower-case? or what's the rule here? lowercase? hyphenated? are spaces aloowed?